maven-plugin/src/main/java/org/openmrs/maven/plugins/GenerateDistro.java

```diff
@@ -0,0 +1,96 @@
+package org.openmrs.maven.plugins;
+
+import org.apache.maven.plugin.MojoExecutionException;
+import org.apache.maven.plugin.MojoFailureException;
+import org.apache.maven.plugins.annotations.Mojo;
+import org.apache.maven.plugins.annotations.Parameter;
+import org.openmrs.maven.plugins.model.Artifact;
+import org.openmrs.maven.plugins.model.DistroProperties;
+import org.openmrs.maven.plugins.model.Server;
+import org.openmrs.maven.plugins.model.Version;
+import org.openmrs.maven.plugins.utility.DistroHelper;
+import org.openmrs.maven.plugins.utility.PropertiesUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Properties;
+
+@Mojo(name = "generate-distro", requiresProject = false)
+public class GenerateDistro extends AbstractTask {
+
+    private static final String O2_DISTRIBUTION = "2.x Distribution";
+
+    private static final String O3_DISTRIBUTION = "O3 Distribution";
+
+    private static final String GENERATE_DISTRO_PROMPT = "You can generate the distro.properties for the following distributions";
```

Comment:
Can we add `file` after distro.properties?

Reply:
I added this.